### PR TITLE
Dependabot auto-merge

### PR DESCRIPTION
**Pull request by Dreamsorcerer**
All remaining PRs on aiohttp-jinja2 auto-merged successfully, so looks like this is the solution.

### CODE REVIEW

**Review by webknjaz — APPROVED** [review]

File: .github/workflows/auto-merge.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+on: pull_request_target
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
```
Comment (webknjaz) [inline .github/workflows/auto-merge.yml:11]: Strictly speaking, if-clauses do not need interpolation syntax and should only contain expressions.

Comment (Dreamsorcerer) [reply .github/workflows/auto-merge.yml:11]: Tell that to Microsoft. :P
That's a verbatim copy/paste from the docs.
https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/automating-dependabot-with-github-actions#enable-auto-merge-on-a-pull-request

**Comment by webknjaz** [comment]
@Dreamsorcerer this patch merges the PRs with the GH-provided token. As a result, merge commits don't produce updates and the CI doesn't run after the merge.
The consequence is that this means that the CI also doesn't upload the coverage report to Codecov. Meaning that Codecov shows a drop in coverage on each of such merges: https://app.codecov.io/gh/aio-libs/aiohttp/branch/master/commits?page=1.

**Comment by Dreamsorcerer** [comment]
OK, this suggests using a personal access token:
https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#triggering-new-workflows-using-a-personal-access-token

**Comment by Dreamsorcerer** [comment]
Although I'm curious that those merges seem to have a pre-commit status check still run against them.

**Pull request merged**